File: angularclient/src/app/rank.ts
import {GameRank} from "./game-rank";

export class Rank {
  constructor(playerName: String, gameRank: GameRank) {
    this.playerName = playerName;
    this.gameRank = gameRank;
  }
  public playerName: String;
  public gameRank: GameRank;

}

File: src/main/java/com/github/floriangubler/eilemitweile/entity/Rank.java
package com.github.floriangubler.eilemitweile.entity;

import com.github.floriangubler.eilemitweile.entity.enumeration.GameRank;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Rank {
    private String playerName;
    private GameRank gameRank;
}
